run-test.sh

```diff
@@ -48,7 +48,7 @@ usage()
     echo "    --restrict-proj <regex>       Run test projects that match regex"
```

Comment:
FYI, I expect this script to be deprecated shortly.  But don't alter the PR on that account.